invenio_app_ils/records/api.py

```diff
@@ -408,9 +416,23 @@ def get_patron(patron_pid):
         if not patron_pid:
             raise PatronNotFoundError(patron_pid)
 
+        if str(patron_pid) == "-1":
+            return SystemPatron()
+
         return Patron(patron_pid)
 
 
+class SystemPatron(Patron):
```

Comment:
nit: `SystemAgent` maybe as a name, maybe others have ideas, @ntarocco, @topless ? We can also leave it like this.

Reply:
Agree, @KonstantinaStoikou I would also add:
```
SystemAgent(Patron):
  id = -1
  ...
```
so that you can replace all occurrences of:
```
SystemPatron().dumps()["id"]
```
with
```
str(SystemAgent.id)
```